libs/remix-core-plugin/src/lib/compiler-metadata.ts

```diff
@@ -26,6 +26,10 @@ export class CompilerMetadata extends Plugin {
     return this.joinPath(path, this.innerPath, contractName + '_metadata.json')
   }
 
+  _OutputFileName (path, contractName) {
+    return this.joinPath(path, this.innerPath, contractName + '_fullOP.json')
```

Comment:
hmm ok, this will be the compilation output per contract.
What do you think of having:
 - <contract_name>.json for the deployment configuration.
 - <contract_name>_metadata.json compilation metadata for this contract.
 - output.json the full compilation result regardless of the contract.

Reply:
if it will be `regardless of the contract.` how will we uniquely identify it?

Reply:
the full compilation result contains the data for all the contracts, we can easily get data for a specific contract, and we store in a filesystem an artefact that is actually pretty common in the ecosystem.